plugins/generator-1.15.2/forge-1.15.2/mappings/types.yaml

```diff
@@ -8,4 +8,5 @@ string: String
 boolean: boolean
 direction: Direction
 advancement: Advancement
-dimensiontype: int
\ No newline at end of file
+dimensiontype: int
+ctx: CommandContext<CommandSource>
```

Comment:
Call this commandcontext or cmdcontext so it is more descriptive. Ctx could be many things